Manager/PostManager.php

```diff
@@ -114,4 +114,19 @@ public function getThumbnailPath(Post $post)
 
         return $post->getGuid();
     }
+    
+    /**
+     * @param \DateTime $date
+     * @return array
+     */
+    public function findByDate(\DateTime $date)
```

Comment:
@eko will review this PR, but I think we should accept ``$date = null`` and set it to current date if the value is null.